### PR TITLE
LPS-129395

### PR DESCRIPTION
**Pull request by martamedio**
cc/ @csierra and @stian-sigvartsen

### CODE REVIEW

**Comment by martamedio** [comment]
ci:test:sf

File: modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -46,6 +46,7 @@ delete-refresh-token=Delete Refresh Token
 details=details
```
Comment (liferay-appsec) [inline modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/content/Language.properties:46]: Check that added properties are either titles or sentences

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by martamedio** [comment]
ci:forward

**Comment by martamedio** [comment]
ci:test:relevant

**Comment by martamedio** [comment]
ci:forward

**Pull request closed without merging**